lib/commands/add-library.ts

```diff
@@ -0,0 +1,28 @@
+///<reference path="../.d.ts"/>
+"use strict";
+
+import path = require("path");
+import Future = require("fibers/future");
```

Comment:
I prefer lower-case syntax as well, but it seems capital-letter syntax is already widely used in the project so I sticked up with the existing convention.